docs/hosted-orchestration/architecture.md

```diff
@@ -0,0 +1,23 @@
+---
+id: architecture
+title: Hosted Orchestration Architecture
+sidebar_label: Architecture
+---
+
+This document provides an overview of the Hosted Orchestration architecture to assist customer engineering, network, and security engineering teams to better understand how it works.
+
+## Architecture Components
+
+From the Sauce Labs side, Hosted Orcheastration includes the following components:
+
+- Component 1
+- Component 2
+- Component 3
+
+Here is an overview of how these components interact with the user environment:
+
+<!-- <img src={useBaseUrl('img/sauce-connect/sc-arch-components.png')} alt="Hosted Orchestration components interacting with user’s environment" width="800"/> -->
+
+### Component 1
+
+Lorem Ipsum description
```

Comment:
Lorem ipsum.

Reply:
I'm guessing you meant this as a placeholder.

Reply:
this is resolved in the latest commits from dan